[TITLE] Feat: 타이틀 씬

voo247 opened the pull request:
## #️⃣PR 구분

- 신규 기능 추가 : #3 
- 기타

## 📝작업 내역 [WIP]
- [x] 타이틀 씬 임시 리소스 교체
- [x] Info 창 구현
- [x] BGM 추가

## 추가 사항
- [ ] info 창 내용 입력
- [x] 오디오 창 다른 씬에도 적용 필요

voo247 committed:
Feat: 타이틀 화면 및 설정창 구현

- 타이틀 로고 애니메이션 추가
- 정보 팝업창 구현
- 소리설정 팝업창 프리팹 등록
- AudioManager를 통한 mute, unmuted 설정 구현
- 효과음 리소스 추가
- 일부 코드 리팩토링

voo247 committed:
Chore: meta 파일 업데이트 및 임시 리소스 제거

- StartStoryScene 스크립트 경로 수정
- meta 정보 업데이트
- 임시 리소스 제거

voo247 committed:
Chore: 스크립트 파일 디렉토리 정리

- Popup 관리 중복 스크립트 삭제 및 병합
- 스크립트 파일 디렉토리 수정
- TMP 폰트 리소스 glow 값 제거

H8bubble commented:
브랜치 충돌나서 머지가 안되는데 git cmd 열고 작업하신 로컬 디렉토리로 이동하셔서 
git pull origin main
git checkout feature/title
git merge main
명령어 입력하시면 cmd창에서 어떤 파일에서 충돌났는지 알려줄거예요 그럼 해당파일 열고 수정해주시면 됩니다

voo247 committed:
Feat: 오디오 설정 기능 구현

- 오디오 세팅 팝업창 프리팹 등록
- 버튼 연결
- 모든 씬에 적용함

voo247 committed:
Feat: 대사 씬 아이템 등장 이벤트 추가

- 대사 씬 더러운 스노우볼 등장 이벤트 추가
- Fix: DialogueParser 잘못된 코드 수정

voo247 commented:
> 브랜치 충돌나서 머지가 안되는데 git cmd 열고 작업하신 로컬 디렉토리로 이동하셔서
> git pull origin main
> git checkout feature/title

아직 타이틀 손볼 게 있어서 머지는 좀 이따가 하려고 합니다! 잠시만 기다려주세요!

voo247 committed:
Merge branch 'main' into feature/title

voo247 committed:
Feat: 추가 씬 오디오 설정 기능 적용

voo247 committed:
Rename: StoryScene > StartStory

voo247 committed:
Refactor: DialogueManager 코드 일부 수정

- 대화씬 종료 시 미니게임 MAIN 씬으로 이동하도록 수정
- s_shake 이벤트에 대한 대기 시간 조정
- Substring 함수 인덱스 형태로 간략화

voo247 committed:
Chore: 빌드 세팅 씬 추가
